message_definitions/v1.0/messages.xml

```diff
@@ -1997,7 +1999,24 @@
     <field name="imageBuffer" type="uint8[]"/>
   </message>
 
-  <!--230 is free -->
+  <message name="SWARMNN" id="230">
```

Comment:
Can you please add descriptions (for the message as well as the fields) and add units?